src/Flicking.ts

```diff
@@ -535,6 +550,94 @@ class Flicking extends Component {
     return this.viewport.remove(index, deleteCount);
   }
 
+  /**
+   * Synchronize info of panels instance with info given by external rendering.
+   * @ko 외부 렌더링 방식에 의해 입력받은 패널의 정보와 현재 플리킹이 갖는 패널 정보를 동기화한다.
+   * @param diffInfo - Info object of how panel elements are changed.<ko>패널의 DOM 요소들의 변경 정보를 담는 오브젝트.</ko>
+   * @param {HTMLElement[]} [diffInfo.list] - DOM elements list after update.<ko>업데이트 이후 DOM 요소들의 리스트</ko>
+   * @param {number[][]} [diffInfo.maintained] - Index tuple array of DOM elements maintained. Formatted with `[before, after]`.<ko>변경 전후에 유지된 DOM 요소들의 인덱스 튜플 배열. `[이전, 이후]`의 형식을 갖고 있어야 한다.</ko>
+   * @param {number[]} [diffInfo.added] - Index array of DOM elements added to `list`.<ko>`list`에서 추가된 DOM 요소들의 인덱스 배열.</ko>
+   * @param {number[]} [diffInfo.removed] - Index array of DOM elements removed from previous element list.<ko>이전 리스트에서 제거된 DOM 요소들의 인덱스 배열.</ko>
+   */
+  public sync(diffInfo: {
+    list: HTMLElement[],
+    maintained: number[][],
+    added: number[],
+    removed: number[],
+  }) {
+    const { list, maintained, added, removed } = diffInfo;
```

Comment:
@WoodNeck 

How about using `changed`?

if (!added.length && !removed.length && !changed.length)  {
    return this;
}
```

Reply:
I'll do it right away~